src/jdk.jdwp.agent/share/native/libjdwp/threadControl.c

```diff
@@ -976,16 +738,9 @@ commonSuspendByNode(ThreadNode *node)
     if (error == JVMTI_ERROR_NONE) {
         node->toBeResumed = JNI_TRUE;
     }
+    JDI_ASSERT(error != JVMTI_ERROR_THREAD_SUSPENDED);
```

Comment:
Should the original code remain in place because another JVMTI agent may have suspended the thread?